reckless_lib/src/utils.rs

```diff
@@ -1,4 +1,5 @@
-use std::{env, fs::create_dir_all, path::Path};
+use git2;
+use std::{env, fmt::format, fs::create_dir_all, path::Path};
```

Comment:
```suggestion
use std::fmt::format;
use std::env;
use std::fs::create_dir_all;
use std::path::Path;
```

It is not a rule, it is just my preference

Reply:
Done.